packages/react/ds/src/card/card.tsx

```diff
@@ -0,0 +1,67 @@
+import React, { cloneElement, ReactElement } from 'react';
+import { Heading } from '../heading/heading.js';
+import { Link } from '../link/link.js';
+import { Paragraph } from '../paragraph/paragraph.js';
+
+type Action = {
+  href: string;
+  text: string;
+};
+
+export type CardProps = {
+  type: 'vertical' | 'horizontal';
+  title?: string;
+  href?: string;
+  img?: string;
+  icon?: ReactElement;
```

Comment:
can we do here `ReactElement<IconPropTypes>` ? it ensures we are passing the right props